Libplanet.Explorer.Tests/Queries/TransactionQueryTest.cs

```diff
@@ -82,6 +83,46 @@ public async Task BindSignatureWithSystemAction()
         Assert.Equal(tx.Serialize(true), ParseHex((string)resultDict["bindSignature"]));
     }
 
+    [Fact]
+    public async Task NextNonce()
+    {
+        async Task AssertNextNonce(long expected, Address address) {
+            var result = await ExecuteQueryAsync(@$"
+            {{
+                nextNonce(
+                    address: ""{address}""
+                )
+            }}
+            ", _queryGraph, source: _source);
+
+            Assert.Null(result.Errors);
+            ExecutionNode resultData = Assert.IsAssignableFrom<ExecutionNode>(result.Data);
+            IDictionary<string, object> resultDict =
+                Assert.IsAssignableFrom<IDictionary<string, object>>(resultData!.ToValue());
+            Assert.Equal(_source.BlockChain.GetNextTxNonce(address), (long)resultDict["nextNonce"]);
+            Assert.Equal(expected, (long)resultDict["nextNonce"]);
+        }
+
+        var key1 = new PrivateKey();
+        await AssertNextNonce(0, key1.ToAddress());
+
+        _source.BlockChain.MakeTransaction(key1, ImmutableList<NullAction>.Empty.Add(new NullAction()));
+        await _source.BlockChain.MineBlock(new PrivateKey());
+        await AssertNextNonce(1, key1.ToAddress());
+
+        _source.BlockChain.MakeTransaction(key1, ImmutableList<NullAction>.Empty.Add(new NullAction()));
+        await _source.BlockChain.MineBlock(new PrivateKey());
+        await AssertNextNonce(2, key1.ToAddress());
+
+        var key2 = new PrivateKey();
+        await AssertNextNonce(0, key2.ToAddress());
+
+        _source.BlockChain.MakeTransaction(key2, ImmutableList<NullAction>.Empty.Add(new NullAction()));
+        await _source.BlockChain.MineBlock(new PrivateKey());
+        await AssertNextNonce(1, key2.ToAddress());
+        await AssertNextNonce(2, key1.ToAddress());
```

Comment:
I think It'll be better to test staged transactions also because `BlockChain<T>.GetTxNonce()` includes staged transactions' nonce.